shipyard-controller/handler/uniformintegrationhandler.go

```diff
@@ -109,7 +109,7 @@ func (rh *UniformIntegrationHandler) Register(c *gin.Context) {
 		}
 
 		raw := fmt.Sprintf("%s-%s-%s-%s-%s", integration.Name, integration.MetaData.KubernetesMetaData.Namespace, integration.Subscription.Filter.Project, integration.Subscription.Filter.Stage, integration.Subscription.Filter.Service)
-		hasher := sha1.New() //nolint:gosec
+		hasher := sha256.New() //nolint:gosec
```

Comment:
The sha1 algorithm is not used in a security sensitive context here, but just to compute a hash. Pls ignore this with `// NOSONAR`